File: TP2DPBO2025C2/php/Kode Program/Baju.php
<?php

require "Aksesoris.php";

class Baju extends Aksesoris {
    private $untuk;
    private $size;
    private $merk;

    public function __construct($ID, $Nama, $Kategori, $Harga, $Gambar, $jenis, $bahan, $warna, $untuk, $size, $merk) {
        parent::__construct($ID, $Nama, $Kategori, $Harga, $Gambar, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

     // Getter methods
     public function getUntuk() {
        return $this->untuk;
    }

    public function getSize() {
        return $this->size;
    }

    public function getMerk() {
        return $this->merk;
    }

    // Setter methods
    public function setUntuk($untuk) {
        $this->untuk = $untuk;
    }

    public function setSize($size) {
        $this->size = $size;
    }

    public function setMerk($merk) {
        $this->merk = $merk;
    }

    public function __destruct() {
    }

}

File: TP2DPBO2025C2/cpp/Kode Program/Baju.cpp
#include "Aksesoris.cpp"

using namespace std;

class Baju : public Aksesoris {
    private:
        string untuk;
        string size;
        string merk;

    public:
        // Konstruktor default
        Baju() {}

        Baju(string untuk, string size, string merk) {
            this->untuk = untuk;
            this->size = size;
            this->merk = merk;
        }

        Baju(string id, string nama, string harga, int stok, string jenis, string bahan, string warna, string untuk, string size, string merk)
        : Aksesoris(id, nama, harga, stok, jenis, bahan, warna) { 
        this->untuk = untuk;
        this->size = size;
        this->merk = merk;
        }

        // Getter
        string getUntuk(){
            return untuk;
        }

        string getSize(){
            return size;
        }

        string getMerk(){
            return merk;
        }

        // Setter
        void setUntuk(string untuk) {
            this->untuk = untuk;
        }

        void setSize(string size) {
            this->size = size;
        }

        void setMerk(string merk) {
            this->merk = merk;
        }

        // Destructor
        ~Baju() {}
};
